Src/BlazorApp/Pages/EndRace.razor

```diff
@@ -1,20 +1,20 @@
 @page "/EndRace"
+
 @using BlazorApp.Services
-@using BlazorApp.Models
 @using BlazorApp.Dtos
+
 @inject IParticipantService Service
 @inject IRaceService RaceService
 
 
-
 <h1>End Race</h1>
 
 <h3>Participants</h3>
 <div>Participants on this race is listed below. Click stop to end the race for the participant.</div>
 
-<table>
+<table class="table">
```

Comment:
@terjetopland I guess you wanted a BootStrap table [https://getbootstrap.com/docs/4.0/content/tables/](https://getbootstrap.com/docs/4.0/content/tables/)

Reply:
Nice :-)